### PR TITLE
fix too much log issue

### PR DESCRIPTION
**Pull request by ldpliu**
https://issues.redhat.com/browse/ACM-5096

### CODE REVIEW

**Comment by ldpliu** [comment]
> I think we should control the log level by parameter. Avoid reducing the log by changing all the levels of the log info.
> 
> What do you think of it?

Yes, you are right.
Changed. https://github.com/stolostron/multicluster-global-hub/pull/655/files#diff-f7893cd19dbd21bb6025ab8336c0fa1799f30e7cfcd17fd0cbcb73c7c0d1e3b9R253
Please review again.

File: operator/pkg/controllers/addon/manifests/templates/agent/multicluster-global-hub-agent-deployment.yaml [hunk 1]
```diff
@@ -22,7 +22,8 @@ spec:
           image: {{ .HoHAgentImage }}
           imagePullPolicy: {{ .ImagePullPolicy }}
           args:
-            - '--zap-devel=true'
+            - --zap-devel=true
```
Comment (morvencao) [inline operator/pkg/controllers/addon/manifests/templates/agent/multicluster-global-hub-agent-deployment.yaml:25]: we may consider to disable zap dev mode to reduce the logs.

**Review by clyang82 — APPROVED** [review]

**Pull request merged**